source/uwp/Renderer/lib/ImageLoadTracker.cpp

```diff
@@ -56,23 +45,18 @@ namespace AdaptiveCards::Rendering::Uwp
         m_hasFailure = true;
 
         // And then notify this image is done
-        /*ComPtr<IBitmapImage> localBitmapImage(bitmapImage);
-        ComPtr<winrt::Windows::Foundation::IInspectable> inspectableBitmapImage;
-        THROW_IF_FAILED(localBitmapImage.As(&inspectableBitmapImage));
-        ImageLoadResultReceived(inspectableBitmapImage.Get());*/
         ImageLoadResultReceived(bitmapImage);
     }
 
     void ImageLoadTracker::AbandonOutstandingImages()
     {
-        // TODO: change to lock instead of try_lock
-        m_lock.try_lock();
+        // TODO: will scoped lock work here?
```

Comment:
scoped_lock is mostly used with multiple mutexes, unqiue_lock should be fine here..